### ISSUE TITLE
conda update conda wants to remove anaconda_navigator (amongst others)

### ISSUE THREAD

**Issue by Dixit22** [conda/conda#9009]
Yesterday (2019-07-26) I updated my base Anaconda environment with 

`conda update anaconda`

which included an update of `conda` from 4.6.14 to 4.7.10. Subsequently whenever I try to update virtually **anything** within this environment I get, in particular: 

`conda update conda -n base -c default`

```
Collecting package metadata (repodata.json): done
Solving environment: \ 
Warning: >10 possible package resolutions (only showing differing packages):
  - https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::gmp-6.1.2-hb37e062_1, https://repo.continuum.io/pkgs/main/osx-64::snappy-1.1.7-he62c110_3, https://repo.continuum.io/pkgs/main/osx-64::unixodbc-2.3.7-h1de35cc_0
  - defaults::gmp-6.1.2-hb37e062_1, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::snappy-1.1.7-he62c110_3, https://repo.continuum.io/pkgs/main/osx-64::unixodbc-2.3.7-h1de35cc_0
  - defaults::snappy-1.1.7-he62c110_3, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::gmp-6.1.2-hb37e062_1, https://repo.continuum.io/pkgs/main/osx-64::unixodbc-2.3.7-h1de35cc_0
  - defaults::gmp-6.1.2-hb37e062_1, defaults::snappy-1.1.7-he62c110_3, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::unixodbc-2.3.7-h1de35cc_0
  - defaults::snappy-1.1.7-he62c110_3, defaults::unixodbc-2.3.7-h1de35cc_0, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::gmp-6.1.2-hb37e062_1
  - defaults::gmp-6.1.2-hb37e062_1, defaults::snappy-1.1.7-he62c110_3, defaults::unixodbc-2.3.7-h1de35cc_0, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3
  - defaults::unixodbc-2.3.7-h1de35cc_0, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::gmp-6.1.2-hb37e062_1, https://repo.continuum.io/pkgs/main/osx-64::snappy-1.1.7-he62c110_3
  - defaults::gmp-6.1.2-hb37e062_1, defaults::unixodbc-2.3.7-h1de35cc_0, https://repo.continuum.io/pkgs/main/osx-64::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::snappy-1.1.7-he62c110_3
  - defaults::gettext-0.19.8.1-h15daf44_3, defaults::unixodbc-2.3.7-h1de35cc_0, https://repo.continuum.io/pkgs/main/osx-64::gmp-6.1.2-hb37e062_1, https://repo.continuum.io/pkgs/main/osx-64::snappy-1.1.7-he62c110_3
  - defaults::gettext-0.19.8.1-h15daf44_3, https://repo.continuum.io/pkgs/main/osx-64::gmp-6.1.2-hb37e062_1, https://repo.continuum.io/pkgs/main/osx-64::snappy-1.1.7-he62c110_3, https://repo.continuum.io/pkgs/main/osx-64::unixodbc-2.3.7-h1de35cc_0
  ... and othedone

## Package Plan ##

  environment location: /Users/Dick/anaconda

  added / updated specs:
    - conda


The following packages will be REMOVED:

  anaconda-navigator-1.9.7-py37_0
  blaze-0.11.3-py37_0
  chest-0.2.3-py37_1
  conda-env-2.6.0-1
  configobj-5.0.6-py37_1
  datashape-0.5.4-py37_1
  dill-0.2.9-py37_0
  flask-cors-3.0.7-py37_0
  odo-0.5.1-py37_0
  pyasn1-0.4.5-py_0
  redis-3.2.0-0
  redis-py-3.2.1-py_0
  sockjs-tornado-1.0.6-py37_0
  tbb-2019.1-h04f5b5a_0
  typed-ast-1.3.4-py37h1de35cc_0
```

Note that I had not run `conda update conda` before `conda update anaconda` because that wanted to downgrade anaconda `2019.03-py37_0 --> custom-py37_1`

Note also that the mention of `repodata.json` in the `Collecting package metadata` line and the `possible package resolution` `Warnings` only appeared after anaconda had been updated `2019.03-py37_0 --> 2019.07-py37_0` 
 
This only applies to the base (anaconda) env. That is, `conda update --all -n anyotherenv` works as expected.  It seems to me that however screwed an environment is `conda update conda` should not try to remove anaconda-navigator.

`conda info`

 ```
    active environment : base
    active env location : /Users/Dick/anaconda
            shell level : 1
       user config file : /Users/Dick/.condarc
 populated config files : /Users/Dick/.condarc
          conda version : 4.7.10
    conda-build version : 3.17.8
         python version : 3.7.3.final.0
       virtual packages : 
       base environment : /Users/Dick/anaconda  (writable)
           channel URLs : https://conda.anaconda.org/anaconda-fusion/osx-64
                          https://conda.anaconda.org/anaconda-fusion/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/Dick/anaconda/pkgs
                          /Users/Dick/.conda/pkgs
       envs directories : /Users/Dick/anaconda/envs
                          /Users/Dick/.conda/envs
               platform : osx-64
             user-agent : conda/4.7.10 requests/2.22.0 CPython/3.7.3 Darwin/18.6.0 OSX/10.14.5
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

`conda config --show-sources`

```
==> /Users/Dick/.condarc <==
pip_interop_enabled: False
ssl_verify: True
channels:
  - anaconda-fusion
  - defaults
```

`conda list --show-channel-urls`

```
# packages in environment at /Users/Dick/anaconda:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                   py37_0    defaults
anaconda                  2019.07                  py37_0    defaults
anaconda-client           1.7.2                    py37_0    defaults
anaconda-navigator        1.9.7                    py37_0    defaults
anaconda-project          0.8.3                      py_0    defaults
appnope                   0.1.0                    py37_0    defaults
appscript                 1.1.0            py37h1de35cc_0    defaults
asn1crypto                0.24.0                   py37_0    defaults
astroid                   2.2.5                    py37_0    defaults
astropy                   3.2.1            py37h1de35cc_0    defaults
atomicwrites              1.3.0                    py37_1    defaults
attrs                     19.1.0                   py37_1    defaults
babel                     2.7.0                      py_0    defaults
backcall                  0.1.0                    py37_0    defaults
backports                 1.0                        py_2    defaults
backports.os              0.1.1                    py37_0    defaults
backports.shutil_get_terminal_size 1.0.0                    py37_2    defaults
beautifulsoup4            4.7.1                    py37_1    defaults
bitarray                  0.9.3            py37h1de35cc_0    defaults
bkcharts                  0.2                      py37_0    defaults
blas                      1.0                         mkl    defaults
blaze                     0.11.3                   py37_0    defaults
bleach                    3.1.0                    py37_0    defaults
blinker                   1.4                      py37_0    defaults
blosc                     1.16.3               hd9629dc_0    defaults
bokeh                     1.2.0                    py37_0    defaults
boto                      2.49.0                   py37_0    defaults
bottleneck                1.2.1            py37h1d22016_1    defaults
bzip2                     1.0.8                h1de35cc_0    defaults
ca-certificates           2019.5.15                     0    defaults
certifi                   2019.6.16                py37_0    defaults
cffi                      1.12.3           py37hb5b8e2f_0    defaults
chardet                   3.0.4                    py37_1    defaults
chest                     0.2.3                    py37_1    defaults
click                     7.0                      py37_0    defaults
cloudpickle               1.2.1                      py_0    defaults
clyent                    1.2.2                    py37_1    defaults
colorama                  0.4.1                    py37_0    defaults
conda                     4.7.10                   py37_0    defaults
conda-build               3.17.8                   py37_0    defaults
conda-env                 2.6.0                         1    defaults
conda-package-handling    1.3.11                   py37_0    defaults
configobj                 5.0.6                    py37_1    defaults
contextlib2               0.5.5                    py37_0    defaults
cryptography              2.7              py37ha12b0ac_0    defaults
curl                      7.65.2               ha441bb4_0    defaults
cycler                    0.10.0                   py37_0    defaults
cython                    0.29.12          py37h0a44026_0    defaults
cytoolz                   0.10.0           py37h1de35cc_0    defaults
dask                      2.1.0                      py_0    defaults
dask-core                 2.1.0                      py_0    defaults
datashape                 0.5.4                    py37_1    defaults
dbus                      1.13.6               h90a0687_0    defaults
decorator                 4.4.0                    py37_1    defaults
defusedxml                0.6.0                      py_0    defaults
dill                      0.2.9                    py37_0    defaults
distributed               2.1.0                      py_0    defaults
docutils                  0.14                     py37_0    defaults
entrypoints               0.3                      py37_0    defaults
et_xmlfile                1.0.1                    py37_0    defaults
expat                     2.2.6                h0a44026_0    defaults
fastcache                 1.1.0            py37h1de35cc_0    defaults
filelock                  3.0.12                     py_0    defaults
flask                     1.1.1                      py_0    defaults
flask-cors                3.0.7                    py37_0    defaults
freetype                  2.9.1                hb4e5f40_0    defaults
get_terminal_size         1.0.0                h7520d66_0    defaults
gettext                   0.19.8.1             h15daf44_3    defaults
gevent                    1.4.0            py37h1de35cc_0    defaults
glib                      2.56.2               hd9629dc_0    defaults
glob2                     0.7                        py_0    defaults
gmp                       6.1.2                hb37e062_1    defaults
gmpy2                     2.0.8            py37h6ef4df4_2    defaults
greenlet                  0.4.15           py37h1de35cc_0    defaults
h5py                      2.9.0            py37h3134771_0    defaults
hdf5                      1.10.4               hfa1e0ec_0    defaults
heapdict                  1.0.0                    py37_2    defaults
html5lib                  1.0.1                    py37_0    defaults
icu                       58.2                 h4b95b61_1    defaults
idna                      2.8                      py37_0    defaults
imageio                   2.5.0                    py37_0    defaults
imagesize                 1.1.0                    py37_0    defaults
importlib_metadata        0.17                     py37_1    defaults
intel-openmp              2019.4                      233    defaults
ipykernel                 5.1.1            py37h39e3cac_0    defaults
ipython                   7.6.1            py37h39e3cac_0    defaults
ipython_genutils          0.2.0                    py37_0    defaults
ipywidgets                7.5.0                      py_0    defaults
isort                     4.3.21                   py37_0    defaults
itsdangerous              1.1.0                    py37_0    defaults
jbig                      2.1                  h4d881f8_0    defaults
jdcal                     1.4.1                      py_0    defaults
jedi                      0.13.3                   py37_0    defaults
jinja2                    2.10.1                   py37_0    defaults
joblib                    0.13.2                   py37_0    defaults
jpeg                      9b                   he5867d9_2    defaults
json5                     0.8.4                      py_0    defaults
jsonschema                3.0.1                    py37_0    defaults
jupyter                   1.0.0                    py37_7    defaults
jupyter_client            5.3.1                      py_0    defaults
jupyter_console           6.0.0                    py37_0    defaults
jupyter_core              4.5.0                      py_0    defaults
jupyterlab                1.0.2            py37hf63ae98_0    defaults
jupyterlab_server         1.0.0                      py_0    defaults
keyring                   18.0.0                   py37_0    defaults
kiwisolver                1.1.0            py37h0a44026_0    defaults
krb5                      1.16.1               hddcf347_7    defaults
lazy-object-proxy         1.4.1            py37h1de35cc_0    defaults
libarchive                3.3.3                h786848e_5    defaults
libcurl                   7.65.2               h051b688_0    defaults
libcxx                    4.0.1                hcfea43d_1    defaults
libcxxabi                 4.0.1                hcfea43d_1    defaults
libedit                   3.1.20181209         hb402a30_0    defaults
libffi                    3.2.1                h475c297_4    defaults
libgfortran               3.0.1                h93005f0_2    defaults
libiconv                  1.15                 hdd342a3_7    defaults
liblief                   0.9.0                h2a1bed3_2    defaults
libpng                    1.6.37               ha441bb4_0    defaults
libsodium                 1.0.16               h3efe00b_0    defaults
libssh2                   1.8.2                ha12b0ac_0    defaults
libtiff                   4.0.10               hcb84e12_2    defaults
libxml2                   2.9.9                hf6e021a_1    defaults
libxslt                   1.1.33               h33a18ac_0    defaults
llvm-openmp               4.0.1                hcfea43d_1    defaults
llvmlite                  0.29.0           py37h98b8051_0    defaults
locket                    0.2.0                    py37_1    defaults
lxml                      4.3.4            py37hef8c89e_0    defaults
lz4-c                     1.8.1.2              h1de35cc_0    defaults
lzo                       2.10                 h362108e_2    defaults
markupsafe                1.1.1            py37h1de35cc_0    defaults
matplotlib                3.1.0            py37h54f8f79_0    defaults
mccabe                    0.6.1                    py37_1    defaults
mistune                   0.8.4            py37h1de35cc_0    defaults
mkl                       2019.4                      233    defaults
mkl-service               2.0.2            py37h1de35cc_0    defaults
mkl_fft                   1.0.12           py37h5e564d8_0    defaults
mkl_random                1.0.2            py37h27c97d8_0    defaults
mock                      3.0.5                    py37_0    defaults
more-itertools            7.0.0                    py37_0    defaults
mpc                       1.1.0                h6ef4df4_1    defaults
mpfr                      4.0.1                h3018a27_3    defaults
mpmath                    1.1.0                    py37_0    defaults
msgpack-python            0.6.1            py37h04f5b5a_1    defaults
multipledispatch          0.6.0                    py37_0    defaults
nbconvert                 5.5.0                      py_0    defaults
nbformat                  4.4.0                    py37_0    defaults
ncurses                   6.1                  h0a44026_1    defaults
networkx                  2.3                        py_0    defaults
nltk                      3.4.4                    py37_0    defaults
nose                      1.3.7                    py37_2    defaults
notebook                  6.0.0                    py37_0    defaults
numba                     0.44.1           py37h6440ff4_0    defaults
numexpr                   2.6.9            py37h7413580_0    defaults
numpy                     1.16.4           py37hacdab7b_0    defaults
numpy-base                1.16.4           py37h6575580_0    defaults
numpydoc                  0.9.1                      py_0    defaults
oauthlib                  3.0.1                      py_0    defaults
odo                       0.5.1                    py37_0    defaults
olefile                   0.46                     py37_0    defaults
openpyxl                  2.6.2                      py_0    defaults
openssl                   1.1.1c               h1de35cc_1    defaults
packaging                 19.0                     py37_0    defaults
pandas                    0.24.2           py37h0a44026_0    defaults
pandoc                    2.2.3.2                       0    defaults
pandocfilters             1.4.2                    py37_1    defaults
parso                     0.5.0                      py_0    defaults
partd                     1.0.0                      py_0    defaults
path.py                   12.0.1                     py_0    defaults
pathlib2                  2.3.4                    py37_0    defaults
patsy                     0.5.1                    py37_0    defaults
pcre                      8.43                 h0a44026_0    defaults
pep8                      1.7.1                    py37_0    defaults
pexpect                   4.7.0                    py37_0    defaults
pickleshare               0.7.5                    py37_0    defaults
pillow                    6.1.0            py37hb68e598_0    defaults
pip                       19.1.1                   py37_0    defaults
pkginfo                   1.5.0.1                  py37_0    defaults
pluggy                    0.12.0                     py_0    defaults
ply                       3.11                     py37_0    defaults
prometheus_client         0.7.1                      py_0    defaults
prompt_toolkit            2.0.9                    py37_0    defaults
psutil                    5.6.3            py37h1de35cc_0    defaults
ptyprocess                0.6.0                    py37_0    defaults
py                        1.8.0                    py37_0    defaults
py-lief                   0.9.0            py37h1413db1_2    defaults
pyasn1                    0.4.5                      py_0    defaults
pycodestyle               2.5.0                    py37_0    defaults
pycosat                   0.6.3            py37h1de35cc_0    defaults
pycparser                 2.19                     py37_0    defaults
pycrypto                  2.6.1            py37h1de35cc_9    defaults
pycurl                    7.43.0.3         py37ha12b0ac_0    defaults
pyflakes                  2.1.1                    py37_0    defaults
pygments                  2.4.2                      py_0    defaults
pyjwt                     1.7.1                    py37_0    defaults
pylint                    2.3.1                    py37_0    defaults
pyodbc                    4.0.26           py37h0a44026_0    defaults
pyopenssl                 19.0.0                   py37_0    defaults
pyparsing                 2.4.0                      py_0    defaults
pyqt                      5.9.2            py37h655552a_2    defaults
pyrsistent                0.14.11          py37h1de35cc_0    defaults
pysocks                   1.7.0                    py37_0    defaults
pytables                  3.5.2            py37h5bccee9_1    defaults
pytest                    5.0.1                    py37_0    defaults
pytest-arraydiff          0.3              py37h39e3cac_0    defaults
pytest-astropy            0.5.0                    py37_0    defaults
pytest-doctestplus        0.3.0                    py37_0    defaults
pytest-openfiles          0.3.2                    py37_0    defaults
pytest-remotedata         0.3.1                    py37_0    defaults
python                    3.7.3                h359304d_0    defaults
python-dateutil           2.8.0                    py37_0    defaults
python-libarchive-c       2.8                     py37_11    defaults
python.app                2                        py37_9    defaults
pytz                      2019.1                     py_0    defaults
pywavelets                1.0.3            py37h1d22016_1    defaults
pyyaml                    5.1.1            py37h1de35cc_0    defaults
pyzmq                     18.0.0           py37h0a44026_0    defaults
qt                        5.9.7                h468cd18_1    defaults
qtawesome                 0.5.7                    py37_1    defaults
qtconsole                 4.5.1                      py_0    defaults
qtpy                      1.8.0                      py_0    defaults
readline                  7.0                  h1de35cc_5    defaults
redis                     3.2.0                         0    https://repo.continuum.io/pkgs/free
redis-py                  3.2.1                      py_0    defaults
requests                  2.22.0                   py37_0    defaults
requests-oauthlib         1.2.0                      py_0    defaults
rope                      0.14.0                     py_0    defaults
ruamel_yaml               0.15.46          py37h1de35cc_0    defaults
scikit-image              0.15.0           py37h0a44026_0    defaults
scikit-learn              0.21.2           py37h27c97d8_0    defaults
scipy                     1.3.0            py37h1410ff5_0    defaults
seaborn                   0.9.0                    py37_0    defaults
send2trash                1.5.0                    py37_0    defaults
setuptools                41.0.1                   py37_0    defaults
simplegeneric             0.8.1                    py37_2    defaults
singledispatch            3.4.0.3                  py37_0    defaults
sip                       4.19.8           py37h0a44026_0    defaults
six                       1.12.0                   py37_0    defaults
snappy                    1.1.7                he62c110_3    defaults
snowballstemmer           1.9.0                      py_0    defaults
sockjs-tornado            1.0.6                    py37_0    defaults
sortedcollections         1.1.2                    py37_0    defaults
sortedcontainers          2.1.0                    py37_0    defaults
soupsieve                 1.8                      py37_0    defaults
sphinx                    2.1.2                      py_0    defaults
sphinxcontrib             1.0                      py37_1    defaults
sphinxcontrib-applehelp   1.0.1                      py_0    defaults
sphinxcontrib-devhelp     1.0.1                      py_0    defaults
sphinxcontrib-htmlhelp    1.0.2                      py_0    defaults
sphinxcontrib-jsmath      1.0.1                      py_0    defaults
sphinxcontrib-qthelp      1.0.2                      py_0    defaults
sphinxcontrib-serializinghtml 1.1.3                      py_0    defaults
sphinxcontrib-websupport  1.1.2                      py_0    defaults
spyder                    3.3.6                    py37_0    defaults
spyder-kernels            0.5.1                    py37_0    defaults
sqlalchemy                1.3.5            py37h1de35cc_0    defaults
sqlite                    3.29.0               ha441bb4_0    defaults
statsmodels               0.10.0           py37h1d22016_0    defaults
sympy                     1.4                      py37_0    defaults
tbb                       2019.1               h04f5b5a_0    defaults
tblib                     1.4.0                      py_0    defaults
terminado                 0.8.2                    py37_0    defaults
testpath                  0.4.2                    py37_0    defaults
tk                        8.6.8                ha441bb4_0    defaults
toolz                     0.10.0                     py_0    defaults
tornado                   6.0.3            py37h1de35cc_0    defaults
tqdm                      4.32.1                     py_0    defaults
traitlets                 4.3.2                    py37_0    defaults
tweepy                    3.7.0                    pypi_0    pypi
typed-ast                 1.3.4            py37h1de35cc_0    defaults
unicodecsv                0.14.1                   py37_0    defaults
unixodbc                  2.3.7                h1de35cc_0    defaults
urllib3                   1.24.2                   py37_0    defaults
wcwidth                   0.1.7                    py37_0    defaults
webencodings              0.5.1                    py37_1    defaults
werkzeug                  0.15.4                     py_0    defaults
wheel                     0.33.4                   py37_0    defaults
widgetsnbextension        3.5.0                    py37_0    defaults
wrapt                     1.11.2           py37h1de35cc_0    defaults
wurlitzer                 1.0.2                    py37_0    defaults
xlrd                      1.2.0                    py37_0    defaults
xlsxwriter                1.1.8                      py_0    defaults
xlwings                   0.15.8                   py37_0    defaults
xlwt                      1.3.0                    py37_0    defaults
xz                        5.2.4                h1de35cc_4    defaults
yaml                      0.1.7                hc338f04_2    defaults
zeromq                    4.3.1                h0a44026_3    defaults
zict                      1.0.0                      py_0    defaults
zipp                      0.5.1                      py_0    defaults
zlib                      1.2.11               h1de35cc_3    defaults
zstd                      1.3.7                h5bba6e5_0    defaults
```

**Comment by Dixit22** [comment]
Today (06-Sep-2019) I still have the issue reported above, so I have tried to reinstall Anaconda from `Anaconda3-2019.07-MacOSX-x86_64.sh`. Perhaps not surprisingly, having done so `conda update conda` updated conda from `4.7.10-py37_0 --> 4.7.11-py37_0`.  Rather more surprisingly when I then tried `conda update anaconda` I got (`conda update --all` gave a similar response):
```
The environment is inconsistent, please check the package plan carefully

The following packages are causing the inconsistency:

  - defaults/osx-64::anaconda==2019.07=py37_0
  - defaults/osx-64::numba==0.44.1=py37h6440ff4_0
```
```
The following NEW packages will be INSTALLED:

  _anaconda_depends  pkgs/main/osx-64::_anaconda_depends-2019.03-py37_0
  fsspec             pkgs/main/noarch::fsspec-0.4.0-py_0
  tbb                pkgs/main/osx-64::tbb-2019.4-h04f5b5a_0

The following packages will be UPDATED:
```
[49 packages]
```
The following packages will be DOWNGRADED:

  anaconda                                   2019.07-py37_0 --> custom-py37_1
```
It seems incredible to me that having just installed something, to bring it to the latest version ("update" it) it should be "downgraded".  At the least my confidence in Anaconda as a package manager is shaken.  What am I doing wrong?  Am I expecting too much?

Since my original post got no comments, I presume I'm posting in the wrong place.  Could someone at least suggest where I should be posting now?

**Comment by msarahan** [comment]
From https://docs.anaconda.com/anaconda/install/update-version/#more-information :

> In the case of the Anaconda metapackage, when you say conda update ipython but you have Anaconda 2019.03, conda can and should “downgrade” Anaconda to the “custom” version so that iPython can be updated.

Your original issue is a duplicate of https://github.com/conda/conda/issues/8842

**Comment by Dixit22** [comment]
Thanks @msarahan , for both re-directing my original issue and for the reference to the Anaconda documentation - I had sadly failed in finding either of them when I searched.

Having studied this documentation at length and having done some more experiments, I believe now I understand just what's going on. Starting from a fresh `anaconda` installation:

When I run 
```
conda update --all
```
I'm quite happy that a number of packages should be updated (those for which new versions have been released since `anaconda 2019.07-py37_0` was created on 2019-07-21 14:43:16): my concern arose because `anaconda` itself was being "downgraded" to `custom-py37_1` - if it said it was being "upgraded" to `custom-py37_1`, I (and I think many others) should have been quite content.

On the other hand when I ran the 
```
conda update conda
conda update anaconda
```
sequence, I didn't understand why any packages (apart from `conda` itself) should be updated - after all I've just installed the latest version of `anaconda` haven't I?  Finally the penny dropped!  `conda` is a component of `anaconda`, and if that is updated (as it was from `4.7.10-py37_0 --> 4.7.11-py37_0`) the "purity" of the `2019.07-py37_0` `anaconda` metapackage is destroyed and `conda` tries to update everything as if I had run `conda update --all`.

Am I right?  Should `conda` really be considered a component of `anaconda`?

Incidentally:

1. you have to run `conda update anaconda` or `conda update --all` twice to get to a fully updated state
2. there are small differences between `conda update anaconda` and `conda update --all`: in the later case, `python` is updated to 3.7.4, `conda-build` is updated to 3.18.9 and `ripgrep` is added.
